pulp.spec

```diff
@@ -409,6 +409,7 @@ Pulp provides replication, access, and accounting for software repositories.
 %{_bindir}/pulp-manage-db
 %{_bindir}/pulp-qpid-ssl-cfg
 %{_bindir}/pulp-gen-ca-certificate
+%{_bindir}/pulp_streamer
```

Comment:
The file is generated by setuptools at build time. This declares that this package (which I just noticed is the server and not the streamer, oops) owns that file. The contents of the file (generated by `python-setuptools-18.0.1-2.fc23.noarch`) are:
```
#!/bin/python
# EASY-INSTALL-ENTRY-SCRIPT: 'pulp-streamer==2.8.0b5','console_scripts','pulp_streamer'
__requires__ = 'pulp-streamer==2.8.0b5'
import sys
from pkg_resources import load_entry_point

if __name__ == '__main__':
    sys.exit(
        load_entry_point('pulp-streamer==2.8.0b5', 'console_scripts', 'pulp_streamer')()
    )
```
I'm not crazy about the use of `#!/bin/python` instead of `#!/usr/bin/env python2`. I believe this problem should be addressed upstream, though. I will see if this has been fixed in the current rawhide release. If not, I'll file a bug.